e2e/package.json

```diff
@@ -0,0 +1,15 @@
+{
+  "name": "e2e",
+  "private": true,
+  "description": "e2e tests using Playwright",
+  "version": "1.0.0",
+  "scripts": {
+    "inst": "yarn playwright install",
+    "ci": "npx playwright test",
+    "start": "npx playwright test",
+    "record": "npx playwright codegen https://hds.hel.fi"
```

Comment:
Is there a reason to use yarn and sometimes npx? Yarn is usually used in our projects.

Reply:
npx is used in playwright documentation

https://playwright.dev/docs/intro#installing-playwright